### ISSUE TITLE
8chan/endchan feature suggestion

### ISSUE THREAD

**Issue by doonoo** [Mishiranu/Dashchan#145]
Can you make it so we can choose the default page to be "user boards" instead of "default boards"? 

Thank you for fixing the app

**Comment by Mishiranu** [comment]
Yes, I've already planned to get rid of "general boards" in some extensions like 8chan-like/endchan/nullchan/etc. Maybe in next version.
